bzkanban.js

```diff
@@ -1362,6 +1362,7 @@ function showBugModal(bugCurrent, bugUpdate) {
     meta.className = 'bug-meta';
 
     body.appendChild(comments);
+    showSpinner();
```

Comment:
Thoughts on making it a cursor instead?

https://css-tricks.com/almanac/properties/c/cursor/ (progress or wait cursor)